modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -26,7 +33,70 @@ metadata:
+
+[source,terminal]
+----
+<timestamp>|<message_serial>|acl_log(ovn_pinctrl0)|<severity>|name="<acl_name>", verdict="<verdict>", severity="<severity>", direction="<direction>": <flow>
+----
+
+Where:
+
+* `<timestamp>` states the time and date for the creation of a log message.
+* `<message_serial>` lists the serial number for a log message.
+* `acl_log(ovn_pinctrl0)` is a literal string that prints the location of the log message in the OVN-Kubernetes plugin.
+* `<severity>` sets the severity level for a log message. If you enable audit logging that supports `allow` and `deny` tasks then two severity levels show in the log message output.
+* `<name>` states the name of the ACL-logging implementation in the OVN Network Bridging Database (`nbdb`) that was created by the network policy.
+* `<verdict>` can be either `allow` or `drop`.
+* `<direction>` can be either `to-lport` or `from-lport` to indicate that the policy was applied to traffic going to or away from a pod.
+* `<flow>` details packet information in a format similiar to the `OpenFlow` protocol. This parameter comprises Open vSwitch (OVS) fields.
+
+The following example shows OVS fields that the `flow` parameter uses to extract packet information from system memory:
+
+[source,terminal]
+----
+<proto>,vlan_tci=0x0000,dl_src=<src_mac>,dl_dst=<source_mac>,nw_src=<source_ip>,nw_dst=<target_ip>,nw_tos=<tos_dscp>,nw_ecn=<tos_ecn>,nw_ttl=<ip_ttl>,nw_frag=<fragment>,tp_src=<tcp_src_port>,tp_dst=<tcp_dst_port>,tcp_flags=<tcp_flags>
+----
+
+Where:
+
+* `<proto>` states the protocol. Valid values are `tcp` and `udp`.
+* `vlan_tci=0x0000` states the VLAN header as `0` because VLAN a ID is not set for internal pod network traffic.
+* `<src_mac>` specifies the source for the Media Access Control (MAC) address.
+* `<source_mac>` specifies the destination for the MAC address.
+* `<source_ip>` lists the source IP address
+* `<target_ip>` lists the target IP address.
+* `<tos_dscp>` states Differentiated Services Code Point (DSCP) values to classify and prioritize certain network traffic over other traffic.
+* `<tos_ecn>` states Explicit Congestion Notification (ECN) values that indicate any congested traffic in your network.
+* `<ip_ttl>` states the Time To Live (TTP) information for an packet.
+* `<fragment>` specifies what type of IP fragments or IP non-fragments to match.
+* `<tcp_src_port>` details the source for the port that is used by both TCP and UDP protocols.
+* `<tcp_dst_port>` lists the destination port for TCP and UDP protocols.
+* `<tcp_flags>` supports numerous flags such as `SYN`, `ACK`, `PSH` and so on. If you need to set multiple values then each value is separated by a vertical bar (`|`). The UDP protocol does not support this parameter.
+
+[NOTE]
+====
+For more information about the previous field descriptions, go to the OVS manual page for `ovs-fields`.
```

Comment:
No, this document is external to Red Hat. It's not something I would want to link to directly as it could change and so forth. Here is the very page: https://man7.org/linux/man-pages/man7/ovs-fields.7.html